### DISCUSSION TITLE
Zoom on heading

### DISCUSSION THREAD

**Discussion by alehandrof** [vslinko/obsidian-outliner#97] [Archive]
I would be interested in the ability to zoom on Markdown headings, as many of my files are a mix of headings and lists. It would work the same way as it does now but show only content from the current heading.

For example, if you zoomed on `## Some heading`, it would show the content from that heading until the next `##` heading, or the end of the file (if there was not other heading).

**Comment by vslinko** [comment]
@alehandrof Thanks for your suggestion!

There is another discussion on headers: https://github.com/vslinko/obsidian-outliner/discussions/53
Please check it. Seems relevant.

I have some thoughts on making another plugin with a zooming feature for headers and lists, but I don't know when I will be able to do that.

**Reply by vslinko** [reply]
Added #99 into my backlog

**Reply by alehandrof** [reply]
Awesome!

**Comment by santiyounger** [comment]
This would be incredible! Happy to know I'm not the only one really needing this! Thanks for the awesome plugin

**Comment by vslinko** [comment]
I made the new Obsidian Zoom plugin with headings support! Please check it out https://github.com/vslinko/obsidian-zoom

**Reply by alehandrof** [reply]
That's awesome!
